calico_node/startup/startup.go

```diff
@@ -34,6 +34,9 @@ import (
 const (
 	DEFAULT_IPV4_POOL_CIDR = "192.168.0.0/16"
 	DEFAULT_IPV6_POOL_CIDR = "fd80:24e2:f998:72d6::/64"
+	AUTOD_METHOD_FIRST     = "first-found"
```

Comment:
hmm `AUTOD_` doesn't sound very intuitive. Can it be something like `AUTODETECT_` or `IFACE_DETECT_` or something similar?